x-pack/plugins/secops/public/components/timeline/data_providers/icons_footer.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiIcon, EuiToolTip } from '@elastic/eui';
+import { isEmpty } from 'lodash/fp';
+import moment from 'moment';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+import { DataProvider } from './data_provider';
+
+const HorizontalBar = styled(EuiHorizontalRule)`
+  margin: 4px 0px;
+`;
+
+const GroupIcons = styled(EuiFlexGroup)`
+  width: 100%;
+`;
+
+interface OwnProps {
+  dataProvider: DataProvider;
+}
+export const IconsFooter = pure<OwnProps>(({ dataProvider }: OwnProps) => {
+  if (!dataProvider.queryDate || (dataProvider.queryDate && isEmpty(dataProvider.queryDate))) {
+    return null;
+  }
+  const dates = dataProvider.queryDate.trim().match(/\d+/g);
+  const tooltipStr = `${moment(parseInt(dates![0], 10)).format('L LTS')} - ${moment(
```

Comment:
This `dates![0]` with the bang is turning off the TypeScript checking which can cause a `TypeError` to show up.

For example if I do:

```ts
const dates = 'a'.trim().match(/\d+/g);
```

dates will be null and then a `dates![0]` will end up throwing a `TypeError`. If we're set on Epochs for most of the app level that could just be a number or an array tuple of two numbers?

As it is, this reads a bit funny if that type is a string that is always two numbers separated by a space. However, if it's always been that way I'm fine with it for the time being.